tests/commons/src/main/java/org/codice/ddf/admin/comp/graphql/GraphQlHelper.java

```diff
@@ -80,6 +90,20 @@ public void waitForGraphQLSchema() {
             });
   }
 
+  public <T> GraphQlResponse<T> mutationRequest(
```

Comment:
Can we rename this to `sendMutationRequest` or something that indicates a request is being sent?